sdk/resourcemanager/azure-resourcemanager-containerinstance/src/test/java/com/azure/resourcemanager/containerinstance/ContainerGroupTest.java

```diff
@@ -225,4 +228,135 @@ public void testBeginCreateWithFileShareVolume() {
+            .define(containerGroupName)
+            .withRegion(region)
+            .withNewResourceGroup(rgName)
+            .withLinux()
+            .withPublicImageRegistryOnly()
+            .withoutVolume()
+            .withContainerInstance("nginx", 80)
+            .withDnsPrefix(dnsNameLbl)
+            .withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy.TENANT_REUSE)
+            .create();
+
+        Assertions.assertTrue(
+            DnsNameLabelReusePolicy.TENANT_REUSE.equals(containerGroup.autoGeneratedDomainNameLabelScope()));
+        containerInstanceManager.containerGroups().deleteById(containerGroup.id());
+
+        // With SubscriptionReuse
+        containerGroup = containerInstanceManager.containerGroups()
+            .define(containerGroupName)
+            .withRegion(region)
+            .withNewResourceGroup(rgName)
+            .withLinux()
+            .withPublicImageRegistryOnly()
+            .withoutVolume()
+            .withContainerInstance("nginx", 80)
+            .withDnsPrefix(dnsNameLbl)
+            .withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy.SUBSCRIPTION_REUSE)
+            .create();
+
+        Assertions.assertTrue(
+            DnsNameLabelReusePolicy.SUBSCRIPTION_REUSE.equals(containerGroup.autoGeneratedDomainNameLabelScope()));
+        containerInstanceManager.containerGroups().deleteById(containerGroup.id());
+
+        // With ResourceGroupReuse
+        containerGroup = containerInstanceManager.containerGroups()
+            .define(containerGroupName)
+            .withRegion(region)
+            .withNewResourceGroup(rgName)
+            .withLinux()
+            .withPublicImageRegistryOnly()
+            .withoutVolume()
+            .withContainerInstance("nginx", 80)
+            .withDnsPrefix(dnsNameLbl)
+            .withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy.RESOURCE_GROUP_REUSE)
+            .create();
+
+        Assertions.assertTrue(
+            DnsNameLabelReusePolicy.RESOURCE_GROUP_REUSE.equals(containerGroup.autoGeneratedDomainNameLabelScope()));
+        containerInstanceManager.containerGroups().deleteById(containerGroup.id());
+    }
+
+    @Test
+    public void testCreateWithBlankDnsNameLabel() {
+        String containerGroupName = generateRandomResourceName("container", 20);
+        String dnsNameLbl = generateRandomResourceName("dnslbl", 20);
+
+        Region region = Region.US_WEST3;
+
+        // With blank DnsNameLabelReusePolicy
+        ContainerGroup containerGroup = containerInstanceManager.containerGroups()
+            .define(containerGroupName)
+            .withRegion(region)
+            .withNewResourceGroup(rgName)
+            .withLinux()
+            .withPublicImageRegistryOnly()
+            .withoutVolume()
+            .withContainerInstance("nginx", 80)
+            .withDnsPrefix(dnsNameLbl)
+            .withAutoGeneratedDomainNameLabelScope(null)
```

Comment:
Remove this line. And combine this test with `testCreateWithDnsNameLabelScopeReuse`

Reply:
Fixed in the [commit#6e748a3](https://github.com/Azure/azure-sdk-for-java/pull/42795/commits/6e748a33351eeda68205b4a03d69db268dd3d0ed)